### ISSUE TITLE
2-Day Timeboxed Design Pass on Docs Section

### ISSUE THREAD

**Issue by ALRubinger** [TBD54566975/developer.tbd.website#439]
I don't have a lot of specificity to provide here. Generally-speaking, the docs section is a bit wonky and could use consistency in fonts, weights, spacing, colors, etc. For instance:

![Image](https://user-images.githubusercontent.com/199891/235844241-3d5cceab-5f58-45f1-9a0d-2783d14b8ff9.png)

There are some greyed-out console calls in the Quickstart:

![Image](https://user-images.githubusercontent.com/199891/235844327-e2bf9eb2-5d6c-4e4e-a4f9-c26aed457361.png)

Some layout on the Docs Home:
![Image](https://user-images.githubusercontent.com/199891/235844389-12e2edca-d224-4a6b-9476-1bd4733a929c.png)

Wildly varying font sizes:
![Image](https://user-images.githubusercontent.com/199891/235844676-1bed61f8-d467-418d-9ce4-af2077422bfd.png)

In summary - readability and visual appeal is important and we're not looking to boil the ocean with a full design, but there's likely a lot we can do with a couple days' investment to align things and and make for a more consistent, pleasing aesthetic. Left to DRI expertise on this one to identify changes we can make in that direction.

**Comment by ALRubinger** [comment]
Goal in @kirahsapong's words: "Comfortable to the eye"

**Comment by ALRubinger** [comment]
Done in ff5429a32fdc912c11312e6eb4053f96b983fe42
